oam/install_scripts/mcs-storagemanager.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=storagemanager
+PartOf=mcs-workernode.service
+Before=mcs-workernode.service
+ConditionPathExists=/etc/columnstore/storagemanager.cnf
+# FailureAction="exit"
+# FailureActionExitStatus=0
+
+[Service]
+Type=simple
+ExecStartPre=@ENGINE_BINDIR@/mcs-start-storagemanager.py
+ExecStart=/usr/bin/env bash -c "LD_PRELOAD=$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}') exec @ENGINE_BINDIR@/StorageManager"
```

Comment:
If you happen to see this PR first, the explanation of the LD_PRELOAD line is in the other PR. We do the same w/ exemgr, primproc, and writeengineserver